### DISCUSSION TITLE
Where to find releases?

### DISCUSSION THREAD

**Discussion by leefernandes** [scalar/scalar#2584] [General]
I'm looking for a list of releases for scalar-api-reference, and am not seeing these documented anywhere obvious.  Anyone know?

**Comment by hanspagel** [comment]
Ah! Does that help?

https://github.com/scalar/scalar/blob/main/packages/api-reference/CHANGELOG.md

**Reply by ShahriarKh** [reply]
Where can I find the one for `nestjs-api-reference`? It's not in the packages.

**Comment by leefernandes** [comment]
That's helpful thanks!  Does scalar release built versions with checksums (to a cdn or other source)?

**Reply by leefernandes** [reply]
nvm, found them on jsdeliver

https://cdn.jsdelivr.net/npm/@scalar/api-reference@1.24.52/dist/browser/standalone.js
